lib/salestation/web/responses.rb

```diff
@@ -29,7 +29,7 @@ class Error < Dry::Struct
         attribute :context, Types::Hash.default({})
 
         def body
-          {message: message}
+          {message: message, debug_message: debug_message}
```

Comment:
What do you think about including `context` as well. This is quite useful when debugging. I guess we might not want this for public endpoints though